jobbergate-api/jobbergate_api/apps/job_scripts/schemas.py

```diff
@@ -89,20 +89,33 @@ class Config:
         schema_extra = job_script_meta_mapper
 
 
-class JobScriptResponse(BaseModel):
+class JobScriptPartialResponse(BaseModel):
     """
     Complete model to match database for the JobScript resource.
+    However, this model does not include job_script_data_as_string from S3.
     """
 
     id: Optional[int] = None
     created_at: Optional[datetime] = None
     updated_at: Optional[datetime] = None
     job_script_name: str
     job_script_description: Optional[str] = None
-    job_script_data_as_string: str
     job_script_owner_email: str
     application_id: int
 
     class Config:
         orm_mode = True
         schema_extra = job_script_meta_mapper
+
+
+class JobScriptResponse(JobScriptPartialResponse):
+    """
+    Complete model to match database in addition to the field
+    job_script_data_as_string from S3, for the JobScript resource.
+    """
+
+    job_script_data_as_string: str
+
+    class Config:
+        orm_mode = True
+        schema_extra = job_script_meta_mapper
```

Comment:
Remove `job_script_data_as_string` from some endpoints to reduce the response size, in line with what we did for applications. It is just included when getting information about specific job script id.

Reply:
@matheushent @dusktreader

Reply:
This approach makes sense for now. We should think more about whether it makes sense to have a separate endpoint for pulling down the script itself (as opposed to its metadata).

Reply:
Yeah, let's keep the idea of a new endpoint to recover the script specifically.

Solving the conversation, no changes are needed for now.